.circleci/config.yml

```diff
@@ -1,183 +1,15 @@
-version: 2
-
-defaults: &defaults
-  working_directory: ~/org
-  # Available images https://hub.docker.com/r/circleci/node/tags/
-
-aliases:
-  - &restore_yarn_cache
-    name: Restore yarn cache
-    keys:
-      - yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
-      - yarn-packages-{{ .Branch }}
-      - yarn-packages-
-  - &save_yarn_cache
-    name: Save yarn cache
-    paths:
-      - ~/.cache/yarn
-    key: yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
-  - &filter_master
-    branches:
-      only: master
-  - &attach
-    at: ~/project
+version: 2.1
```

Comment:
Why an empty workflow?

Reply:
Because the circleci flow is required. We can switch that to the Build action once this is merged :)